Dalamud/Interface/Internal/Windows/PluginInstallerWindow.cs

```diff
@@ -76,9 +75,8 @@ internal class PluginInstallerWindow : Window, IDisposable
         private List<AvailablePluginUpdate> pluginListUpdatable = new();
         private bool hasDevPlugins = false;
 
-        private bool downloadingIcons = false;
-        private Dictionary<string, (bool IsDownloaded, TextureWrap?[]? Textures)> pluginImagesMap = new();
-        private Dictionary<string, (bool IsDownloaded, TextureWrap? Texture)> pluginIconMap = new();
+        private Dictionary<string, TextureWrap?> pluginIconMap = new();
```

Comment:
No need to save IsDownloaded now. If there is no value, it needs to be downloaded and a placeholder is inserted beforehand. If there is a value, its either null/failed, or a valid tex.